### DISCUSSION TITLE
devtunnel.rp Cask auto update PR error

### DISCUSSION THREAD

**Discussion by setaskin** [Homebrew/discussions#5520] [Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

We set up a release pipeline that was creating an auto pr via brew bump-cask-pr .  Recently we started to get this below error. I saw devtunnels added to autobump.txt.  Are we supposed to be on this list ?, I am a bit confused we only update version via release pipeline. CAn you explain what has changed and how can we fix it ? 
<img width="551" alt="image" src="https://github.com/user-attachments/assets/8784bb0f-26c5-4708-84f8-07882ab85f8d">

### Relevant casks

Casks/d/devtunnel.rb

**Comment by setaskin** [comment]
@bevanjkay

**Reply by bevanjkay** [reply]
The version bumps have been managed by the autobump workflow in `homebrew-cask` for some time now. Whenever a new version is published to `https://tunnelsassetsprod.blob.core.windows.net/cli/cli-version.json` a PR is opened by @BrewTestBot when the workflow runs (once every 3 hours).

**Reply by p-linnane** [reply]
@setaskin If you would prefer to open PR's via your release workflow, we can definitely remove that cask from the autobump list. Just let us know.

**Reply by setaskin** [reply]
@bevanjkay , @p-linnane , thank you for quick response. I wasn't aware of this change. We can stay in autobump list. That sounds good. I will remove our pipeline homebrew stage than. Thanks for your help
